lnbits/core/views/api.py

```diff
@@ -27,13 +28,21 @@ def api_payments():
 @api_validate_post_request(
     schema={
         "amount": {"type": "integer", "min": 1, "required": True},
-        "memo": {"type": "string", "empty": False, "required": True},
+        "memo": {"type": "string", "empty": False, "required": False},
+        "description_hash": {"type": "string", "empty": False, "required": False},
     }
 )
 def api_payments_create_invoice():
+    if "description_hash" in g.data:
```

Comment:
Shouldn't we check if at least one of the two parameters (`memo` or `description_hash`) has been included?

Reply:
Can't we have `memo` as an empty string if nothing is provided?

Reply:
I forgot to add `"default": ""` to `memo`.